src/content/about-the-data/file-f.mdx

```diff
@@ -1,8 +1,9 @@
 import { AboutTheDataMarkdownLink } from '../../js/components/aboutTheDataSidebar/AboutTheDataMarkdownLink.jsx';
 
 File F (Subaward Attributes) contains data about subawards. These
-data are submitted by prime award recipients to the [FFATA Subaward Reporting System
-(FSRS)](https://www.fsrs.gov/). New data come to USAspending.gov by
+data are submitted by prime award recipients to the
+ <a target="_blank" rel="noopener noreferrer" href="https://www.fsrs.gov/">FFATA Subaward Reporting System(FSRS</a>).
```

Comment:
I just happened to notice a typo in this line while spot-checking links.
We need a space before the opening paren.